src/com/example/TicTacToe.java

```diff
@@ -1,9 +1,46 @@
 package com.example;
 
 public class TicTacToe {
+    private static int cx,co;
+    private static int win[][]={{0, 1, 2},{3, 4, 5},{6, 7, 8},{0, 3, 6},{1, 4, 7},{2, 5, 8},{0, 4, 8},{2, 4, 6}};
+    private static void Count(char[] ch){
+        for (char aCh : ch) {
+            if (aCh == 'x') {
+                cx++;
+            } else if (aCh == 'o') {
+                co++;
+            }
+        }
+    }
+    private static boolean playerWin(char[] board, char c)
+    {
+        for (int i=0; i<8; i++)
```

Comment:
You aren't using brackets, always use brackets even if you are only using one line. You use 8, thats a magic number. Magic numbers are generally not good.